app/views/govuk_publishing_components/components/docs/checkboxes.yml

```diff
@@ -123,6 +123,22 @@ examples:
           value: "green"
         - label: "Blue"
           value: "blue"
+  with_custom_size_for_page_header_and_caption:
+    description: |
+      If a custom heading size is passed in, the page heading and caption size will be changed to set size while still retaining a heading level.
```

Comment:
I'd suggest we agree on a description based on https://github.com/alphagov/govuk_publishing_components/pull/1616#discussion_r459408223 and make this the same.

Reply:
Updated to match the radio and removed additional example from docs